---
title: "The Galaxy Europe team at the ELIXIR All Hands Meeting 2024"
date: "2024-06-10"
tease: "The Galaxy Europe team has participated in the ELIXIR All Hands meeting 2024 in Uppsala, Sweden."
hide_tease: false
supporters:
  - eurosciencegateway
  - unifreiburg
  - elixir
subsites: [global, freiburg, eu, esg]
tags: [esg,elixir]
main_subsite: eu
---
The [10th ELIXIR All Hands meeting](https://elixir-events.eventscase.com/EN/ahm2024) was held in Uppsala, Sweden, from June 10 to 12, 2024. The Galaxy Europe team attended this meeting,
which aimed to bring together members of the ELIXIR community from across the ELIXIR Nodes, as well as collaborators from partner organisations, to review ELIXIR's achievements and activities and discuss future plans.

In the [Deciphering federated analysis: What is it and how far it can take us](https://docs.google.com/document/d/13aCXWW8gnv9AYvPUDyOLO_oBQYacsW3EpJ9IKk5OzbM/) session on the first day, the participants discussed the following question. How can we conduct collaborative analysis while respecting data privacy and following data protection regulations? Multiple ELIXIR initiatives are working on federated analysis, which will solve this problem. The Galaxy Europe team, as part of the [EuroScienceGateway project](https://galaxyproject.org/projects/esg/), was present at this gathering. Dr. Björn Grüning presented [How Galaxy Can Be Used for Federated Analytics?](https://docs.google.com/presentation/d/1PBMeVux-ukXoCsj8mAaUf_c3oHWCYwgm5cpc37WTkY4/edit#slide=id.g2e3832b6d94_1_48) during this meeting. He discussed how the Galaxy architecture is similar to EUCAIM, as well as Pulsar Nodes, ELIXIR AAI, and the GA4GH TES Cluster, and the use of distributed nodes for various types of efficiency, including carbon efficiency. He also addressed the storage problem for federated data analysis and the Galaxy team's solutions, which include connecting to and selecting multiple public or personal storage alternatives via API, command line, and GUI, in addition to the temporarily expanding users' scratch space. Pulsar provides a varied collection of alternatives, including S3 storage from several countries, commercial S3, OneData, iRODS, and Rucio. In the same session, Dr. Tim Beck presented [the role of EOSC-ENTRUST in secure federated data analysis](https://docs.google.com/presentation/d/1PBMeVux-ukXoCsj8mAaUf_c3oHWCYwgm5cpc37WTkY4/edit#slide=id.g2e3832b6d94_1_53) and discussed [RO-Crate](https://www.researchobject.org/ro-crate/) and [workflow processing](https://workflowhub.eu/), which enable the interoperability framework and execution approach for practical pan-Trusted Research Environment (TRE) analysis.

On the second day of the event, the [Tackling single-cell data management and interoperability in Galaxy](https://docs.google.com/document/d/16ykadI4A8l4wrviaBunan4ujxU36sJCk5_ZUOZTw_XU/edit?tab=t.0#heading=h.gjdgxs) session discussed the advancements of analysis tool suites and the abundance of data. Poor interoperability of diverse single-cell data types has been identified as a challenge in this field. New tools and regular releases of R and Python tools bring new data formats, many of which lack backward compatibility, posing challenges for tutorial and workflow sustainability. In this session, ELIXIR members explored incorporating ELIXIR data management techniques and how to use available tools such as the ELIXIR single-cell RDM toolkit to assist users in standardizing RDM and data interoperability. Wendi Bacon and Pavankumar Videm [presented](https://docs.google.com/presentation/d/1CM0bFJFuYezVOqNyFsZp2BrsygGRYMs43p1ColbYihI/edit#slide=id.g268ba6ba735_0_0) the lifetime of single-cell data on Galaxy and explored the problems in standards and data formats. Participants in this workshop were divided into groups, led as well by Björn Grüning and Wolfgang Maier ran breakout groups, to create and test workflows and documentation, incorporate public metadata standards into the workflows, and expand the [ELIXIR-RDM toolkit page](https://rdmkit.elixir-europe.org/single_cell_sequencing) applicable standards. The output of these workflows was formatted in the research object crate (RO-Crate) format, which is compliant with ELIXIR specifications.

During the [Where is the dark matter? The Galaxy Community shedding light on activities, connections, scope, and unaddressed shortcomings](https://docs.google.com/document/d/1i5coxz4ubcMdMr5E4v6OVPqbHs2S7ujOVnkRj6I33Cg/edit?tab=t.0), on the second day of the event, the Galaxy Community in ELIXIR explored the prospects Galaxy brings as both a platform and a community. In short, the primary goal of this workshop was to promote knowledge about the wide range of accessible tools, procedures, and projects from closely related domains. This session was organized as a world cafe setup and divided into three major groups:  (a) [Research Data Management in Galaxy](https://docs.google.com/document/d/1cpT4iI5YxavfQ9gsCyCuZ69aL8jNVa8_QdjFrsXNGDc/edit?tab=t.0#heading=h.p0gzezzi3us4) moderated by Dr. Frederik Coppens and Dr. Rafael Andrade Buono (b) [Training and Skills](https://docs.google.com/document/d/1ScpWMWXF8JULMrPfAiMoIs1Ez4DykwzL2iYpRchGNvs/edit?tab=t.0#heading=h.da5gwyeoxhss) moderated by Dr. Bérénice Batut (c) [Galaxy Workflows and the IWC](https://docs.google.com/document/d/1cUhebg99EHICWD6KVhfGDthwZloW4BylZwokrFpPaq4/edit?tab=t.0#heading=h.da5gwyeoxhss) moderated by Dr. Wolfgang Maier.

On the 3rd day of this event, during the [Connecting ELIXIR's technical ambitions](https://docs.google.com/document/d/1d1BgeD3qIJya0dXcB7wTI50QOmnHtJ-E_CfD9Kt3HLs/edit?tab=t.0#heading=h.gjdgxs) session, the Galaxy Community opportunities and advancements were presented in the [Reproducible analytics and infrastructure](https://docs.google.com/presentation/d/1k2bIFHAtxv7C_YaI6Idb9ZWirbiIIvQL/edit#slide=id.g2e2e5863106_0_52) by Dr. Frederik Coppens. During this session, EuroScienceGateway and the Galaxy community's advancements, such as increasing content on the Galaxy Training Network (GTN), as well as infrastructure advancements, such as Bring Your Own Storage (BYOS) and Bring Your Own Compute (BYOC) features, were discussed. The project's future path, which includes lowering environmental impact, community sustainability, federated sensitive data analysis, and growing the array of tools and workflows, was presented to the audience. In addition, the Galaxy Community's ties to other ELIXIR components such as [LS Login](https://lifescience-ri.eu/ls-login/version-2023/user/how-to-log-in-to-the-service-using-your-credentials.html), [Bioschemas](https://bioschemas.org/), [RO-Crate](https://www.researchobject.org/ro-crate/), [WorkflowHub](https://workflowhub.eu/), [ELIXIR bio.tools](https://elixir-europe.org/services/tools/biotools), and [OpenEBench](https://elixir-europe.org/services/openebench) were discussed. Finally, the different ways available for [contributing to and getting involved in](https://galaxyproject.org/community/contributing/) the Galaxy Community has been discussed.

On the third day of the event, during the [ELIXIR Cloud - A roadmap towards operationalisation and expansion](https://docs.google.com/document/d/1iMhd4cjVKOhoTaTDjk616VIuanQyDhgjOak0e3izLAE/edit?tab=t.0) workshop, the members discussed the ELIXIR Cloud, which is a modular and interoperable cloud infrastructure for genome-scale data analysis. This is based on community standards established by the [Global Alliance for Genomics and Health (GA4GH)](https://www.ga4gh.org/). Stian Soiland-Reyes demonstrated [WorkflowHub cloud integration in ELIXIR](https://docs.google.com/presentation/d/1SQGTaBvNxWF5gMrH3e_3Wpl7xX-6Z8Ho/edit#slide=id.g2e15719ef5d_0_103). WorkflowHub standardizes metadata and allows you to link publications, datafiles, workflows, and protocols, as well as execute, publish, and cite workflows. Workflowhub can accommodate a wide range of workflow systems, including CWL, Galaxy, NextFlow, and RO-Crate. It also supports integration with GitHub, GitLab, [Galaxy IWC](https://iwc.galaxyproject.org/), and [Nextflow nf-core](https://nf-co.re/). WorkflowHub also provides the framework for publishing and sharing workflows with DOIs using [DataCite](https://datacite.org/), [Zenodo](https://zenodo.org/), [ORCID](https://orcid.org/), and the [COVID-19 Data Portal](https://www.covid19dataportal.org/). In this session, Björn Grüning demonstrated the Galaxy's [federated storage](https://docs.google.com/presentation/d/1SQGTaBvNxWF5gMrH3e_3Wpl7xX-6Z8Ho/edit#slide=id.g2e15719ef5d_0_124). [Data Repository Service (DRS), Task Execution Service (TES), Beacon APIs as well as Tool Registry Service (TRS)](https://galaxyproject.org/ga4gh/) were presented during this workshop. The challenges of increasing storage requirements, as well as recent Galaxy advancements (e.g., deferred data, temporary additional storage space up to 4 weeks, export to long-term archives, and BYOS), were discussed.

On the final day, multiple participants presented the various capabilities of the ELIXIR Research Software Ecosystem (RSEc) and its application in Life Science research during the [The Research Software Ecosystem in action: examples and use-cases](https://docs.google.com/document/d/1iR6V2ZGd2QSKdgr96CVYW45es5v4vfncqDqGelUoFRU/edit?tab=t.0) seminar. Dr. Bérénice Batut [discussed](https://docs.google.com/presentation/d/1Ao0dA-u0-5EP1c0yvxrbBVuiVtFBlsE3RriJmHszWeA/edit#slide=id.g2e1ad0cf65f_1_56) the latest developments to expose tools and workflows for specific scientific communities, as well as the bi-directional association of tools and training materials. She also demonstrated how user support could be integrated into this package, as well as how Galaxy tools, workflows, training, and support for [EDAM](https://edamontologydocs.readthedocs.io/en/latest/what_is_edam.html) and bio.tools IDs are linked together.
